ikasaneip/rest/rest-module-client/src/main/java/org/ikasan/rest/client/ModuleRestClientAutoConfiguration.java

```diff
@@ -35,4 +35,9 @@ public ModuleControlRestServiceImpl moduleControlRestService(Environment environ
         return new ModuleControlRestServiceImpl(environment);
     }
 
+    @Bean
+    public TriggerRestServiceImpl triggerRestService(Environment environment){
```

Comment:
Why returning an implementation class rather than an interface?

Reply:
@mitcje @andrzej-majewski I am going to approve so i can merge this onto the master. Will create JIRA to address.